File: ios/common/events/withArgs/VerificationStatusEvent.swift
import Foundation

struct VerificationStatusEvent: EventWithArgs {
    var type: String { return "onVerificationStatusReceived" }
    var status: VerificationStatus
    
    func getData() -> [String: String] {
        return ["status": status.rawValue]
    }
    
    enum VerificationStatus: String {
        case ACCEPTED = "ACCEPTED"
        case REJECTED = "REJECTED"
    }
}

File: android/src/main/java/io/mosip/tuvali/common/events/withArgs/VerificationStatusEvent.kt
package io.mosip.tuvali.common.events.withArgs

import io.mosip.tuvali.common.events.Event

data class VerificationStatusEvent(val status: VerificationStatus): Event {

  enum class VerificationStatus(val value: Int) {
    ACCEPTED(0),
    REJECTED(1)
  }
}
